hypothesis-python/tests/ghostwriter/test_expected_output.py

```diff
@@ -176,6 +177,10 @@ def divide(a: int, b: int) -> float:
                 style="unittest",
             ),
         ),
+        # (
+        #     "xml_etree_ElementTree_py38",
+        #     ghostwriter.magic(xml.etree.ElementTree),
+        # ),
```

Comment:
Delete dead code?

Reply:
opps I did it again. Too many long files I was lost lol